src/main/kotlin/racingcar/RacingCarManager.kt

```diff
@@ -0,0 +1,44 @@
+package racingcar
+
+import racingcar.domain.Car
+import racingcar.domain.DriveConditionImpl
+import racingcar.presentation.InputManager
+import racingcar.presentation.OutPutManager
+
+class RacingCarManager(
+    val inputManager: InputManager,
+    val outPutManager: OutPutManager
```

Comment:
```suggestion
    private val inputManager: InputManager,
    private val outPutManager: OutPutManager
```

외부에서 사용하지 않는 프로퍼티는 private으로 변경해주는 건 어떨까요?